examples/loitering/demo.ipynb

```diff
@@ -0,0 +1,356 @@
+    "\n",
+    "Knowing where the person is, the next step is to determine if the coordinate is within the region of interest (i.e. the garage), marked by the red polygon shown below.\n",
+    "\n",
+    "<img src=\"./demo.assets/region.png\" alt=\"region of interest\" style=\"zoom: 30%;\" />\n",
+    "\n",
+    "`vqpy.utils.within_regions([REGION, ...])` accepts a coordinate and returns `True` if it's in any of the `REGION`s.\n",
+    "\n",
+    "The polygon is outlined by its 5 vertices:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "REGION = [(550, 550), (1162, 400), (1720, 720), (1430, 1072), (600, 1073)]"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The predicate to filter VObjs in the region is then:\n",
+    "\n",
+    "```python\n",
+    "\"bottom_center\": vqpy.utils.within_regions([REGION])\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since we wish to find person staying in the region for some duration, `vqpy.utils.continuing` could wrap the above predicate and specify the minimum time it should be satisfied.\n",
+    "\n",
+    "`duration=10` sets the time threshold to 10 seconds; `name=\"in_roi\"` can be used later to retrieve time periods of condition being satisfied.\n",
+    "\n",
+    "Predicate used to filter on `bottom_center` becomes:\n",
+    "\n",
+    "```python\n",
+    "\"bottom_center\": vqpy.utils.continuing(\n",
+    "    condition=vqpy.utils.within_regions([REGION]),\n",
+    "    duration=10, name=\"in_roi\"\n",
+    ")\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Combining everything we have so far, the filter condition of the query should be:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filter_cons = {\n",
+    "    \"__class__\": lambda x: x == Person,\n",
+    "    \"bottom_center\": vqpy.utils.continuing(\n",
+    "        condition=vqpy.utils.within_regions([REGION]), duration=10, name=\"in_roi\"\n",
+    "    ),\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Selecting VObjs' properties for output"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For each person, we can output its tracking id, coordinate, and duration of staying in the region:\n",
+    "\n",
+    "- tracking id can be selected with `track_id`\n",
+    "- outputting coordinate needs to select `coordinate` then serialize the ndarray to string with post-processing function `lambda x: str(x)`\n",
```

Comment:
1. In select cons, the key is a property name, which can be either a property in VObj, or a property in our built-in property library. The value is a post-processing function you want to apply to the property as an output. 
2. maybe explain a little more about the difference between `coordinate` and the `bottom_center` you used in filtering, i.e why you use `bottom_center` in filter and `coordinate` as output.
3. And maybe explain more where the names come from: in vobj/in property lib/introduced by continuing ..  for track_id and coordinate

Reply:
Added more explanation